### ISSUE TITLE
Primary callouts do not change color when selecting Bootswatch themes (in Bootstrap3 template)

### ISSUE THREAD

**Issue by Hitch42** [giterlizzi/dokuwiki-plugin-bootswrapper#103]
### Description

When displaying a callout using the primary color option, selecting other Bootswatch themes does not change its color, including icons.

### Steps to reproduce

1. Install Bootstrap Wrapper Plugin
2. Install Bootstrap3 Template
3. Display a callout using the primary color option.
4. Change the theme using different Bootswatch themes.

**Expected behavior:** The callout border color and icon should be affected by Bootswatch themes, like other primary-colored elements.

**Actual behavior:** The callout border color and icon are not affected by Bootswatch themes, while other primary-colored elements are affected.

### Versions

* Bootstrap3 Template: v2019-05-22-24
* Plugins: Bootstrap Wrapper Plugin 2017-04-07
* DokuWiki: Release 2018-04-22b "Greebo"
* Browser: Chrome and Firefox

**Comment by MvErven** [comment]
Have you had a look at the configuration settings?
Changing theses do change the colors at my site.

![wiki-themes](https://user-images.githubusercontent.com/17900348/60594652-b72f3600-9da5-11e9-8238-ec1e5225d419.png)

**Comment by Hitch42** [comment]
The themes do change the colors, but not for the callouts.

Here's a shot of a test page with a default theme and a Bootswatch Simplex theme.

![](https://i.imgur.com/b3LBlTV.jpg)

![](https://i.imgur.com/klgpVzE.jpg)

**Comment by MvErven** [comment]
I once wanted a purple panel, so I got an CSS line.
In the advanced configuration, I changed the userall.css and added these line. Maybe this may guide you in the right direction. 

`.bs-wrapper.panel-purple .panel {
    border-color: #dddddd;
}
.bs-wrapper.panel-purple .panel-default > .panel-heading {
    background: #7432A5; 
    color: #ffffff;
    border-color: #dddddd;
}
.bs-wrapper.panel-fixed .panel {
    max-height: 200px;
}
.bs-wrapper.panel-fixed .panel-default > .panel-heading {
    max-height: 200px;
}
`
